mythril/laser/ethereum/instructions.py

```diff
@@ -1837,6 +1833,27 @@ def stop_(self, global_state: GlobalState):
         """
         global_state.current_transaction.end(global_state)
 
+    @staticmethod
+    def _write_symbolic_calldata(
```

Comment:
The calldata is something different from the return data right?
In this case you seem to be returning the latter